SparkPost/2web2ui
# FE-1423 Replace account dropdown button & nav link focus styles

### What Changed

- Updates the Account Dropdown menu button
- Updates navigation link focus styles 

### How To Test
- Run the app locally
- Verify the menu button matches the mocks: https://sparkpost.invisionapp.com/console/share/YW2D6G8KX6/516054126
- Verify the change only affects the desktop verion
- Verify navigation link focus styles work

Old vs. New
![Screen Shot 2021-03-31 at 10 28 11 AM](https://user-images.githubusercontent.com/3903325/113160984-f7864700-920b-11eb-8dda-e6f57d27b317.png)
